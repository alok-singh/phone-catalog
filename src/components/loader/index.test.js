import React from "react";
import Loader from "./index";
import { render, cleanup } from "@testing-library/react";

import "@testing-library/jest-dom/extend-expect";

const renderApp = () => {
  return render(<Loader />);
};

beforeEach(() => {});

afterEach(() => {
  cleanup();
});

test("Header renders correctly", () => {
  const { container } = renderApp();
  expect(container.firstChild).toMatchSnapshot();
});
