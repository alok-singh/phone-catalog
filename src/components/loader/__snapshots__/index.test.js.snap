// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  class="loader-wrapper"
>
  <div
    class="app-loader"
  >
    <span />
  </div>
</div>
`;
